docs/UserGuide.md

```diff
@@ -43,144 +42,260 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 
 ### 2.1. Viewing help : `help`
 
-Format: `help`
+Format: `help [COMMAND]`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> * When there is no command specified, `help` will print help messages of all commands.
+> * Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to Dueue<br>
+Format: `add n/TASKNAME [due/DUEDATE] [t/TIME] [#LISTNAME] [d/DESCRIPTION] [@VENUE] [p/PRIORITYLEVEL] [*f]`
 
-> Persons can have any number of tags (including 0)
+> * Tasks cannot have multiple lists.
+> * Tasks must be added with task name.
+> * The required field for a task is TASKNAME, while other fields are optional.
+> * By default, task will be added to current viewing list, unless [#LISTNAME] is specified.
+> * Due dates of tasks can repeating, e.g. every sunday.
+> * Time of each task can only be one time point, e.g. 4pm.
+> * Specifications of fields for task can be entered in any order.
+> * If no fields are entered, help message for add command will appear.
+> * User can choose to replace the priority levels `IMPORTANT`, `NORMAL` and `TRIVIAL` by 3, 2 and 1.
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/laundry due/every sunday #personal d/wash clothes @B1 *f`<br>
+  Add a task with taskname `laundry` due `every sunday` under the list `personal` with description `wash clothes` at the venue `B1` and mark it as favourite.
+* `add n/2103lecture due/every friday t/4pm #study d/go for lecture p/3 @iCube`<br>
+  Add a task with taskname `2103lecture` due `every friday` at time `4pm` under the list `study` with description `go for lecture` at the venue `iCube` with priority level `IMPORTANT`
+
+### 2.3. Listing all tasks : `list`
```

Comment:
2.3 list
- typo in the format? e.g. "list list ListINDEX [favourite]" doesnt make sense for the second "list"?

- it is unclear how the "favourite" works or why it is needed.